sdk-actors/src/test/java/io/dapr/actors/client/DaprGrpcClientTest.java

```diff
@@ -5,114 +5,127 @@
   }
 
   @Test
   public void invoke() {
-    String methodName = "mymethod";
-    byte[] payload = "{ \"id\": 123 }".getBytes();
-    byte[] response = "\"OK\"".getBytes();
-
-    SettableFuture<DaprProtos.InvokeActorResponse> settableFuture = SettableFuture.create();
-    settableFuture.set(DaprProtos.InvokeActorResponse.newBuilder().setData(ByteString.copyFrom(response)).build());
-
-    when(grpcStub.invokeActor(argThat(argument -> {
-      assertEquals(ACTOR_TYPE, argument.getActorType());
-      assertEquals(ACTOR_ID, argument.getActorId());
-      assertEquals(methodName, argument.getMethod());
-      assertArrayEquals(payload, argument.getData().toByteArray());
-      return true;
-    }))).thenReturn(settableFuture);
-    Mono<byte[]> result = client.invoke(ACTOR_TYPE, ACTOR_ID, methodName, payload);
-    assertArrayEquals(response, result.block());
+    Mono<byte[]> result = client.invoke(ACTOR_TYPE, ACTOR_ID_OK, METHOD_NAME, REQUEST_PAYLOAD);
+    assertArrayEquals(RESPONSE_PAYLOAD, result.block());
   }
 
   @Test
   public void invokeNullPayload() {
-    String methodName = "mymethod";
-    byte[] response = "\"OK\"".getBytes();
-
-    SettableFuture<DaprProtos.InvokeActorResponse> settableFuture = SettableFuture.create();
-    settableFuture.set(DaprProtos.InvokeActorResponse.newBuilder().setData(ByteString.copyFrom(response)).build());
-
-    when(grpcStub.invokeActor(argThat(argument -> {
-      assertEquals(ACTOR_TYPE, argument.getActorType());
-      assertEquals(ACTOR_ID, argument.getActorId());
-      assertEquals(methodName, argument.getMethod());
-      assertArrayEquals(new byte[0], argument.getData().toByteArray());
-      return true;
-    }))).thenReturn(settableFuture);
-    Mono<byte[]> result = client.invoke(ACTOR_TYPE, ACTOR_ID, methodName, null);
-    assertArrayEquals(response, result.block());
+    Mono<byte[]> result = client.invoke(ACTOR_TYPE, ACTOR_ID_NULL_INPUT, METHOD_NAME, null);
+    assertArrayEquals(RESPONSE_PAYLOAD, result.block());
   }
 
   @Test
   public void invokeException() {
-    String methodName = "mymethod";
-
-    SettableFuture<DaprProtos.InvokeActorResponse> settableFuture = SettableFuture.create();
-    settableFuture.setException(new ArithmeticException());
-
-    when(grpcStub.invokeActor(argThat(argument -> {
-      assertEquals(ACTOR_TYPE, argument.getActorType());
-      assertEquals(ACTOR_ID, argument.getActorId());
-      assertEquals(methodName, argument.getMethod());
-      assertArrayEquals(new byte[0], argument.getData().toByteArray());
-      return true;
-    }))).thenReturn(settableFuture);
-    Mono<byte[]> result = client.invoke(ACTOR_TYPE, ACTOR_ID, methodName, null);
+    Mono<byte[]> result = client.invoke(ACTOR_TYPE, ACTOR_ID_EXCEPTION, METHOD_NAME, null);
 
     assertThrowsDaprException(
         ExecutionException.class,
         "UNKNOWN",
-        "UNKNOWN: java.lang.ArithmeticException",
+        "UNKNOWN: ",
```

Comment:
This has to do with how Exceptions are handled in the grpc library, it does not show the exception as it would in the blocking path.